src/content/docs/infrastructure/amazon-integrations/manage-aws-data/aws-integrations-metrics.mdx

```diff
@@ -4574,6 +4595,162 @@ We no longer add these mappings, and we're committed to preserving all the new m
+    >
+
+We generate New Relic [entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic) for many AWS namespaces. When we generate New Relic entities for a namespace you can expect to:
+
+* Browse those entities in our [entity explorer](/docs/new-relic-one/use-new-relic-one/ui-data/new-relic-one-entity-explorer).
+* Access an entity dashboard automatically created for those entities.
+* Get metrics and entities from that namespace decorated with AWS tags. Collecting AWS tags requires that you have given New Relic the `tag:GetResources` permission, which is part of the setup process in the UI. AWS tags show in metrics as `tag.AWSTagName`; for example, if you have set a `Team` AWS tag on the resource, it will show as `tag.Team`.
+* Leverage all the features included in the entity explorer.
+
+<Callout variant="important">
+  [New Relic Lookout](/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance) is not compatible with the AWS Metric Streams integration.
+</Callout>
+
+    </Collapser>
+</CollapserGroup>
+
+## Create alert conditions [#set-alerts]
+
+You can create NRQL alert conditions on metrics from a metric stream. Make sure your filter limits data to metrics from CloudWatch Metric Stream only. To do that, construct your queries like this:
+
+```sql
+SELECT sum(aws.s3.5xxErrors) FROM Metric WHERE collector.name = 'cloudwatch-metric-streams' FACET aws.accountId, aws.s3.BucketName
+```
+
+To make sure your alerts process the data correctly, configure advanced signal settings. Advance signal settings addresses how AWS CloudWatch receives metrics from services with a certain delay. For example, Amazon guarantees that 90% of EC2 metrics are available in CloudWatch within 7 minutes of them being generated. When you add streaming metrics from AWS to New Relic, however, it can add up to 1 minute additional delay as data needs to buffer in Amazon Firehose.
+
+To configure the signal settings, go to <DNT>**Condition Settings**</DNT> and click on <DNT>**Advanced Signal Settings**</DNT>. Enter the following values:
+
+    1. <DNT>**Aggregation window**</DNT>. We recommend setting it to 1 minute. If you're having issues with flapping alerts or alerts not triggering, consider increasing it to 2 minutes.
+    1. <DNT>**Offset evaluation by**</DNT>. Depending on the service, CloudWatch may send metrics with a certain delay. The value is set in windows. With a 1-minute aggregation window, setting the offset to 8 ensures the majority of the metrics are evaluated correctly. You may be able to use a lower offset if the delay introduced by AWS and Firehose is less.
+    1. <DNT>**Fill data gaps with**</DNT>. Leave this void, or use Last known value if gaps in the data coming from AWS lead to false positives or negatives.
+
+See our documentation on [how to create NRQL alerts](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/) for more details.
```

Comment:
I think you can delete this and just put the link in the first sentence where you already mention NRQL alert conditions